prac_05/emails.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404/CP5632 Practical - Christian Torrisi
+Do-from-scratch - Gets a users emails and names and stores them in a dictionary
+"""
+
+
+def main():
+
+    email_to_name = {}
+    email = input("What is your email address?\n>>> ")
+
+    while email != "":
+        email_to_name[email] = get_name(email)
+        check_name = input("Is your name {}? (Y/n) ".format(email_to_name[email])).upper()
+        if check_name == "Y" or check_name == "":
```

Comment:
I use this to make it simpler to write, but it still works the way you have done it with OR
if check_name in ["Y", ""]: